docs/IoT_quick_start/Dashboard.md

```diff
@@ -0,0 +1,37 @@
+# Dashboard
+
+The **Dashboard** displays widgets with visualized device data for your domain. Widgets are set and configured in the **Data model** tab of the **Device Center**, along with their corresponding **Observations** to enable collection of the reported device data. The **Dashboard** widgets are displayed per domain and feature different kinds of charts depending on the type of data collected.
+
+!!! important
+    The **Dashboard** and widgets are domain-wide, so they only apply to devices belonging to your domain. You can add up to 10 devices to a single widget, and they are added chronologically based on their **Creation time**. So if there are, for example, 15 devices in your domain, the 10 oldest of them will only be presented on a widget.
+
+![Dashboard menu entry](images/dashboard_view.png "Dashboard menu entry"){:style="float: left;margin-right: 817px;margin-top: 17px;margin-bottom: 17px;border: 1px solid lightgray"}
+
+## Dashboard overview
+
+The **Dashboard** consists of widgets which are based on observations set on a device's data model. If you can't see any widgets in your domain dashboard, follow the [Visualize device data](Visualize_device_data.md) quick start to set them up smoothly.   
+
+  ![Dashboard](images/dashboard.png "Dashboard"){:style="float: left;margin-right: 817px;margin-top: 17px;margin-bottom: 17px;border: 1px solid lightgray;"}
+
+### Widgets
```

Comment:
We should add information that there could be only 15 widgets per each domain. Adding more is prohibited. Remaining widgets count is displayed while adding a new one.

Reply:
note added